test/cucumber/evaluation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: vizualize evaluation
+	 As a teacher
+	 I want to see evaluation
+	 So I can see any details that i want
+
+
+Scenario: vizualize evaluation
+	Given I am on Teaching Assistant Teacher's home page
+	When i click on the Vizualize button
```

Comment:
melhor: When I search for "evaluation1" 
-aumenta a abstração
-reduz o tamanho
-não menciona elementos da user interface ("click")